docs/src/Provider.doc.js

```diff
@@ -19,65 +19,179 @@ card(
     props={[
       {
         name: 'colorScheme',
-        type: `"light" | "dark" | "userPreference"`,
+        type: `'light' | 'dark' | 'userPreference'`,
         defaultValue: 'light',
-        description:
-          'The color scheme for components inside the provider. Specify "userPreference" to use "prefers-color-scheme" media query.',
-        href: 'colorScheme',
+        description: `The color scheme for components inside the provider. Specify 'userPreference' to use 'prefers-color-scheme' media query.`,
```

Comment:
"Use `userPreference` to allow the end user to specify via their browser settings, using the `prefers-color-scheme` media query."

Reply:
DONE